rest-api/alembic/versions/8b1281fb55ed_add_ppi_views.py

```diff
@@ -0,0 +1,63 @@
+"""Add PPI views
+
+Revision ID: 8b1281fb55ed
+Revises: 8d12872e0b77
+Create Date: 2017-09-20 15:39:17.562293
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = '8b1281fb55ed'
+down_revision = '8d12872e0b77'
+branch_labels = None
+depends_on = None
+
+_QUESTIONNAIRE_RESPONSE_ANSWER_VIEW_SQL = """
```

Comment:
Same comment as usual -- please add an overview comment. (Who will care if I change this, where/how is it used?)

Reply:
Added comment.